src/editors/containers/ProblemEditor/data/ReactStateOLXParser.js

```diff
@@ -175,17 +225,27 @@ class ReactStateOLXParser {
   }
 
   buildNumericInput() {
-    const question = this.addQuestion();
     const demandhint = this.addHints();
     const answerObject = this.buildNumericalResponse();
     const problemObject = {
       problem: {
-        ...question,
         [ProblemTypeKeys.NUMERIC]: answerObject,
         ...demandhint,
       },
     };
-    return this.builder.build(problemObject);
+
+    const temporaryBuild = this.builder.build(problemObject);
+
+    const problemType = "problem";
```

Comment:
When we create the numeric type problem in the legacy editor, the question is present inside the `<numericalresponse>` tag. But here we are inserting the question outside it. Is this intentional ?

Reply:
@kaustavb12 I don't know the reason behind this. The previous implementation was doing the same, and test cases were written to match it. If you look at the OLXParser.js, it even has a comment which mentions that numeric input questions are stored in a different place. So in my implementation I tried to match the existing behavior.

Reply:
I now see, that the earlier implementation also had the same behaviour. Going by the comment on OLXParser, this is perhaps done to handle some edge cases.

Perhaps @farhaanbukhsh can confirm this.

Reply:
@kaustavb12 @Cup0fCoffee  This was intentionally done, if you see the test cases for multiple answers the answers generally go under `<numericalresponse>` hence to standardize it we chose to go this way.